src/components/Hero/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Flex, Button, Text } from '@chakra-ui/react'
+
+export const Hero = (): JSX.Element => {
```

Comment:
- Content and media could be provided via props instead of being static.
- A darker layer background could be placed between media and content in order to improve visibility.
- In terms of consistency in design, you might use brand color (`primary`) instead of arbitrary style for `Button`.